PWGCF/Tasks/dptdptcorrelations-simchl.cxx

```diff
@@ -1502,7 +1502,7 @@ struct CheckGeneratorLevelVsDetectorLevel {
     for (auto& part : mcParticles) {
       auto pdgpart = fPDG->GetParticle(part.pdgCode());
       if (pdgpart != nullptr) {
-        float charge = (pdgpart->Charge() / 3 >= 1) ? 1.0 : ((pdgpart->Charge() / 3 <= -1) ? -1.0 : 0.0);
+        float charge = (pdgpart->Charge() >= 3) ? 1.0 : ((pdgpart->Charge() <= -3) ? -1.0 : 0.0);
```

Comment:
I merge it now, but these three lines I would have just written
if (pdgpart->Charge() != 0) {
  ngen++
}
there will be no quarks I assume :)

Reply:
It will not be the first time I've found the quarks at generator level
Thanks anyway for the suggestion
And for the fixes!